tests/TestCase/Controller/Component/RequestHandlerComponentTest.php

```diff
@@ -336,7 +337,7 @@ public function testViewClassMap()
     public function testIsAjaxParams()
     {
         $this->request->env('HTTP_X_REQUESTED_WITH', 'XMLHttpRequest');
-        $event = new Event('Controller.startup', $this->Controller);
+        $event = new ControllerEvent('Controller.startup', $this->Controller);
```

Comment:
It seems redundant still having to use `Controller.` prefix for the event name when you are using `ControllerEvent` class. It should add the prefix internally.